packages/statsd/src/test/client.ts

```diff
@@ -0,0 +1,3 @@
+describe('statsd', () => {
+  it.todo('works');
```

Comment:
I'd like to see some tests here, not sure exactly how we could make them meaningful. Looking at `hot-shots` itself they setup a weird local server to see that it receives the expected requests. :/

I'm ok with them coming in as a subsequent PR at worst.